code/python/scripts/flow.py

```diff
@@ -1,13 +1,31 @@
 #!/usr/bin/env python
 """
-Originally written by Anthony Gitter
-Edited by Chris Magnano for batch use
+Find paths, by solving the min-cost flow problem, through a protein-protein interaction network that connect source proteins in the  <sources_file> with targets in the <targets_file>. 
+
+Usage: flow.py --min-sources 1 --min-targets 1 --edges-file data/edges_file.txt --mapping-file data/mapping_file.txt --sources-file data/hf_curve_shape_ensp_stringdb_filter.txt --targets-file data/gene_lists/mehle_targets_ensp.txt --outdir flow_results
```

Comment:
I'm noticing the description from argparse includes
```
  --outdir OUTDIR       Output file
```
which should probably be "output directory"

What does ` --flow-meta-file` do? Is that an output file that is written?

Reply:
I cleaned this up some. There is still a flow meta file that is written that just contains the objective function value, which is still not that useful because iirc the program rescales the amount of flow depending on the inputs so the final objective function value is not necessarily comparable across runs. But the original intent with the argument was to be able to do iterative runs of flow.py where the n+1 run would use the amount of flow from the previous run but adjust the capacities of edges from the targets to the sink. It was an idea I was playing around with but didn't take that far.